src/PackageWatcher.ts

```diff
@@ -78,14 +78,14 @@ export class PackageWatcher {
      * launch configuration if required and then resolve the package
      * dependencies.
      */
-     async handlePackageSwiftChange() {
```

Comment:
We should look at integrating a linter/formatter for these kind of issues, I'll create an issue

Reply:
I've been running with ESLint, which seems to be the standard Linter. But it doesn't catch these annoying indent issues